extensions/extension.go

```diff
@@ -147,6 +148,17 @@ func initInternal(c *Config) error { //nolint:funlen,gocognit,gocyclo,maintidx /
 	if err != nil {
 		return err
 	}
+	jsonFn = object.Extension{
+		Name:     "json_go",
```

Comment:
Why is "json_go" identifier?

Reply:
not sure... I have `json` where the serialization is done by hand (and uses a few extra tags for functions and errors), this one is using go's marshaller (and the new unwrap)

suggestion for a better name?

Reply:
I wouldn't use json_go as grol shouldn't mention Go

Can you use a to_json, object, array, to_object, json_object, or from_json ?

I'm unsure the usage of the operator
Wording is a pain

Reply:
naming is hard. grol being in go (so far) is a bit part of the language (Go REPL...) but maybe more relevant: the extension mechanism does let you map any go function to grol (and bit like Tcl did with C back in the days)

I want to keep json() maybe I can make this one json_pp (pretty print) but the pretty print is just an option (the additional optional string padding argument)

both `json()` and `json_go()` are functions (not operators btw) - there is `unjson()` which is really... eval() atm but that one maybe could be named `from_json()` instead; that doesn't help me name the 2 `json`s though :)